pkg/apis/batch/validation/validation.go

```diff
@@ -352,6 +356,15 @@ func validateScheduleFormat(schedule string, fldPath *field.Path) field.ErrorLis
 	return allErrs
 }
 
+func validateTimeZone(timeZone string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if _, err := time.LoadLocation(timeZone); err != nil {
```

Comment:
kube-controller-manager main package should import time/tzdata to make sure there's always a fallback for time zone data

xref https://pkg.go.dev/time/tzdata@master

Reply:
this makes the valid values for this API field system-dependent, which I'm not thrilled about... do we know the case-sensitivity of these values, or future stability of these values? is there any chance a value we accept and persist today would be unsupported or stop working in N years? how portable are manifests tested on one system?

do we want to support the "Local" time zone? how do we define that?

Reply:
As far as I understand these are not system-dependent but rather golang dependent, we'll be embedding the golang provided version of TZ data. The stability and updates are defined in https://www.iana.org/go/rfc6557 so I'm rather confident about its consistency.

> do we want to support the "Local" time zone? how do we define that?

No, "Local" should not be allowed, only explicit time zones as defined in https://www.iana.org/time-zones